tests/centos-6-vars.yml

```diff
@@ -0,0 +1 @@
+drupalvm_webserver: nginx
```

Comment:
I was a bit worried of how to override things at first. Then I just tried to see what happens if you re-declare a YAML variable with a different value :smile: No errors!